### PR TITLE
DSY-1006: Updates xcode settings to 11.4 (5.2)

### PR DESCRIPTION
**Pull request by raonivaladares**
# Description

- Update settings to Xcode 11.4

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [x] Internal changes

### CODE REVIEW

**Comment by raonivaladares** [comment]
@viviancrs is it interesting some one of system review this?
like mr.Bueno?  or Rubens?

File: SampleApp/NatDS-SampleApp.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -42,7 +42,7 @@
 		A9D71BCB2386DFF200CDEC1D /* Assets.xcassets in Resources */ = {isa = PBXBuildFile; fileRef = A9D71BCA2386DFF200CDEC1D /* Assets.xcassets */; };
 		A9D71BCE2386DFF200CDEC1D /* LaunchScreen.storyboard in Resources */ = {isa = PBXBuildFile; fileRef = A9D71BCC2386DFF200CDEC1D /* LaunchScreen.storyboard */; };
 		A9D71BD72386E12600CDEC1D /* NatDS.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = A9D71BD62386E12600CDEC1D /* NatDS.framework */; };
-		A9D71BD82386E12600CDEC1D /* NatDS.framework in Embed Frameworks */ = {isa = PBXBuildFile; fileRef = A9D71BD62386E12600CDEC1D /* NatDS.framework */; settings = {ATTRIBUTES = (RemoveHeadersOnCopy, ); }; };
+		A9D71BD82386E12600CDEC1D /* NatDS.framework in Embed Frameworks */ = {isa = PBXBuildFile; fileRef = A9D71BD62386E12600CDEC1D /* NatDS.framework */; settings = {ATTRIBUTES = (CodeSignOnCopy, RemoveHeadersOnCopy, ); }; };
```
Comment (raonivaladares) [inline SampleApp/NatDS-SampleApp.xcodeproj/project.pbxproj:45]: Do CodeSignOnCopy affects us as embed framework?

ps:  I am positive that will not, but not 💯 % sure.

Comment (michelbueno) [reply SampleApp/NatDS-SampleApp.xcodeproj/project.pbxproj:45]: Nope! It won't affect code signing :)

Comment (raonivaladares) [reply SampleApp/NatDS-SampleApp.xcodeproj/project.pbxproj:45]: Gracias! bueno!

**Pull request merged**